server/src/main/java/org/apache/druid/metadata/SqlSegmentsMetadataQuery.java

```diff
@@ -927,7 +929,7 @@ private static void bindColumnValuesToQueryWithInCondition(
       final SQLStatement<?> query
   )
   {
-    if (CollectionUtils.isNullOrEmpty(values)) {
```

Comment:
It is okay to return here on empty as well since if `values` is empty, we won't be doing any binding anyway.

Reply:
True. I just kept it for consistency with its sibling method `getParameterizedInConditionForColumn()`.